vue/src/lib/alert/examples/default.vue

```diff
@@ -4,7 +4,7 @@
       <div class="row">
         <md-button
           ariaLabel="Click to Open"
-          @click="() => show = true"
+          @click="() => show = !show"
```

Comment:
this is for select or should some of these changes be in a separate PR?

Reply:
Reversed. Changed for demo last week.